pkg/controller/releasemanager/incarnation.go

```diff
@@ -857,10 +856,14 @@ func (i *IncarnationCollection) releasable() (r []*Incarnation) {
 		}
 	}
 	if len(r) == 0 && len(i.willRelease()) == 0 {
-		i.controller.getLog().Info("there are no releases, looking for retired release to unretire")
+		i.controller.getLog().Info("no viable releases. looking for releases to take out of retirement")
 		candidates := i.unretirable()
 		if len(candidates) > 0 {
-			candidates[0].fastRelease()
+			firstUnretireable := candidates[0]
```

Comment:
I'm confused about this. Shouldn't fast release make it so it transitions to deploying on the next reconcile loop? Do you understand why this fixes it?

Reply:
I have not tested this yet. I'm planning on running it in sandbox today to see if it actually works. But the problem is that setting `"releaseEligible": true` does not actually get saved on retired releases, so they never transition to `deploying`.